docs/collection/findOneAndUpdate.md

```diff
@@ -8,7 +8,7 @@ Find a document and update it in one atomic operation, requires a write lock for
 
 1. `query` *(String|ObjectId|Object)*
 
-2. `update` *(Object)*: Update operations to be performed on the document
+2. `update` *(Object)*: Update operations to be performed on the document. As [specified in MongoDB Node.js native driver 3.x](https://github.com/mongodb/node-mongodb-native/blob/master/CHANGES_3.0.0.md#updateone--updatemany), you need to specify an atomic operator here (like a `$set`, `$unset`, or `$rename`).
```

Comment:
Could you add a link to `https://docs.mongodb.com/manual/reference/operator/update/` instead of the changelog (which won't be relevant for new users)?

Reply:
good point